elastic/kibana
# [Guided onboarding] Security tour fixes

## Summary

Fixes https://github.com/elastic/cloud/issues/104482 by adding Cypress tests for the onboarding tour.
Fixes https://github.com/elastic/kibana/issues/136846 by updating the copy.
Fixes https://github.com/elastic/kibana/issues/136852 by closing the tour if the "Add integrations" button is clicked. 

### How to test
Most changes are test related and copy updates. To review changes for closing the tour by clicking the "Add integrations" button see this [file](https://github.com/elastic/kibana/pull/136958/files#diff-8b700725072994c477c478744ab1e35a8420606b50a3ad9f6b112a98c0a05f90). 

To test this change: 
1. Navigate to /home#/getting_started and select Security solution. 
2. Click through the tour to the last step that points to the "Add integrations" button.
3. Instead of "End tour" click "Add integrations" that navigates you away from the Security solution. 
4. Navigate back to the Security solution and verify that the tour is not shown anymore. 

### Screencast


https://user-images.githubusercontent.com/6585477/181035771-ba39c2f9-38db-4e58-ad32-c8c05c08ada1.mov
